dask_jobqueue/core.py

```diff
@@ -153,8 +154,8 @@ def start_workers(self, n=1):
         workers = []
         for _ in range(n):
             with self.job_file() as fn:
-                out = self._call([self.submit_command, fn])
-                job = out.decode().split('.')[0]
+                out = self._call(shlex.split(self.submit_command) + [fn])
+                job = out.decode().split('.')[0].strip()
```

Comment:
And by the way the `.strip()` here is necessary because `qsub -terse` output finishes with a `\n`.